kafka-rest/src/test/java/io/confluent/kafkarest/integration/v3/AclsResourceIntegrationTest.java

```diff
@@ -337,6 +337,45 @@ public void testCreateSearchAndMultiDelete() {
         multiDeleteResourceTypeAll.readEntity(DeleteAclsResponse.class));
   }
 
+  @Test
+  public void testMultiDeleteBadQueryParameter() {
```

Comment:
Is the test unrelated to improved exception handling above? Seems so!

Reply:
The test does test the improved exception handling. The NOT_FOUND on 355 is the improvement.